ui/src/app/pages/facility/component/providers/providers.module.ts

```diff
@@ -0,0 +1,61 @@
+import { NgModule } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { ClarityModule } from '@clr/angular';
+import { ReactiveFormsModule,FormsModule } from '@angular/forms';
+import { PagesModule } from '../../../pages.module';
+import { ProvidersRoutingModule } from './providers-routing.module';
+import { ProvidersComponent } from './providers.component';
+import { ProvidersAddComponent } from './providers-add/providers-add.component';
+import { DcimService } from '../dcim/dcim.service';
+@NgModule({
+  imports: [
+    CommonModule,
+    ProvidersRoutingModule,
+    ClarityModule,
+    ReactiveFormsModule,
+    FormsModule,
+    PagesModule,
+   
+  ],
+  declarations: [ProvidersComponent, ProvidersAddComponent],
+
+  providers:[DcimService]
+})
+export class ProvidersModule {
```

Comment:
All these properties of this module should same as properties of data model 'Asset'